applications/nvidia_nim/nvidia_nim_nvclip/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+	"application": {
+		"name": "NV-CLIP NIM",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "Python",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "1.0.3",
+			"tested_versions": [
+				"1.0.3",
+				"2.1.0"
+			]
```

Comment:
2.5.0?

Reply:
Added 2.5.0 and tested both 1.0.3 and 2.1.0